packages/strapi-admin/admin/src/components/Users/ModalCreateBody/utils/loginSettingsForm.js

```diff
@@ -0,0 +1,10 @@
+import RolesSelectComponent from '../../LoginModalSection/RolesSelectComponent';
+
+const loginSettingsForm = {
```

Comment:
Can we rename this utility as it refers to the role's setting?

Reply:
In fact, it refers to the login settings section in the role form. Anyway, I will rename it to roleSettingsForm